### ISSUE TITLE
Can't connect Ledger Nano S to the multisig wallet

### ISSUE THREAD

**Issue by maptuhec** [gnosis/MultiSigWallet#341]
Hey there,

I am trying to connect my ledger nano S with the multisig wallet, but I am facing issues. I have followed all the steps from the instructions page. My ledger has the browser data enabled by default. It says that the connection is successful but I can't see the balance nor interact with the wallet.

I have tried both on the web version and the desktop version.

Device: Ledger Nano S
Firmware version : 1.5 && 1.6 (tried with two devices)

![image](https://user-images.githubusercontent.com/33101661/80713840-c6356080-8afc-11ea-80f6-bbb4de1ff51d.png)

**Comment by denisgranha** [comment]
Hi! for last ledger firmwares I would recommend you ti use ledger through the metamask integration as they handle well all firmware versions and you can select different derivation paths.

**Comment by maptuhec** [comment]
That actually worked, but I don't want to use metamask, as a proxy (I have to teach users to use MM). And this will not work on the desktop version unfortunately.

Thanks, though

**Comment by denisgranha** [comment]
I agree, but if you want proper integration with Ledger with up-to-date Multisg time to switch to Gnosis Safe :) https://gnosis-safe.io/

The problem by updating the ledger libraries is that what fixes for you it would break for users with previous firmwares and most people that use this Multisig haven't touched funds for long time

**Comment by denisgranha** [comment]
To put it in a bit of context. In the past 3 years, ledger changed the derivation path for keys, I personally still use the old ones for this Multisig. 

If we want to support new ledger firmwares directly, there will be a need to configure different derivation paths, etc

**Comment by maptuhec** [comment]
Thanks for the explanation!
